lib/spack/spack/cmd/info.py

```diff
@@ -162,6 +162,11 @@ def print_text_info(pkg):
 
     color.cprint(section_title('Homepage: ') + pkg.homepage)
 
+    if (pkg.maintainers != ''):
+        mnt = " ".join([m.strip() for m in pkg.maintainers.split(",")])
```

Comment:
@alalazo i was trying to have `"@" + m.strip()` here, but got some errors due to colours

> ==> Error: invalid color specifier: 'd' in '@*bMaintainers: @.@davydden @jppelteret

could you please advise how to fix it? I don't know python well enough to understand what's going on.

Reply:
it's certainly related to `plain_format = '@.'` used for color output...

Reply:
nevermind, fixed it

> To output an @, use '@@'.  To output a } inside braces, use '}}'.

from https://github.com/LLNL/cram/blob/master/src/python/llnl/util/tty/color.py#L73